app/javascript/controllers/new_user_version_controller.js

```diff
@@ -6,6 +6,11 @@ export default class extends Controller {
   connect () {
     if (!this.hasUserVersionYesTarget || !this.hasUserVersionNoTarget) return
 
+    // when first loading page, neither version radio is selected
+    if (this.userVersionYesTarget.checked === false && this.userVersionNoTarget.checked === false) {
+      this.fileUploadsFieldsetTarget.disabled = true
+      this.fileSectionTarget.style.opacity = 0.5
```

Comment:
You should just be able to add the `opacity-50` class. See https://getbootstrap.com/docs/5.3/utilities/opacity

Reply:
Cool, thanks for pointing out that's available.